graftm/hmmer.py

```diff
@@ -52,20 +45,23 @@ def hmmalign(self, input_path, directions):
         input_path : str
             Filename of unaligned hits to be aligned
         directions : dict
-            dictionary containing readnames as keys, and compliment direction
-            as the entry (False=Forward, True=Reverse)
+            dictionary containing read names as keys, and compliment
+            as the entry (True=Forward compliment, False=Reverse compliment)
         Returns
         -------
-        Nothing - output files are known.
+        conv_files : array
```

Comment:
No need to say `conv_files`. And while I'm here, to be a bit more specific maybe
```
list of two strings, where each string is a path to an aligned fasta file (forward and reverse files respectively)
```